src/helm/benchmark/static/schema.yaml

```diff
@@ -2954,540 +2984,560 @@ run_groups:
 
-  - name: cleva_fact_checking
-    display_name: CLEVA (Chinese) fact checking
-    description: CLEVA (Chinese) fact checking
+  - name: cleva_pinyin_transliteration
+    display_name: CLEVA (Chinese) pinyin transliteration
+    description: Scenario that asks the model to translate between Chinese and Pinyin.
     metric_groups:
-      - accuracy
-      - calibration
+      - cleva_pinyin_transliteration_metrics
       - general_information
+      - efficiency
     environment:
-      main_name: exact_match
+      main_name: chinese_bleu_1
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
-      language: Chinese
+      task: pinyin transliteration
+      what: n/a
+      who: automatically generated by algorithm
+      when: n/a
+      language: Chinese, Pinyin
 
-  - name: cleva_inductive_reasoning
-    display_name: CLEVA (Chinese) inductive reasoning
-    description: CLEVA (Chinese) inductive reasoning
+  - name: cleva_classical_chinese_understanding
+    display_name: CLEVA (Chinese) classical Chinese understanding
+    description: Scenario for evaluating the understanding of classical Chinese by selecting the appropriate classical Chinese translation for a given modern Chinese sentence.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
-      language: Chinese
+      task: multiple-choice
+      what: n/a
+      who: n/a
+      when: n/a
+      language: Classical Chinese
 
-  - name: cleva_instruction_following
-    display_name: CLEVA (Chinese) instruction following
-    description: CLEVA (Chinese) instruction following
+  - name: cleva_coreference_resolution
+    display_name: CLEVA (Chinese) coreference resolution
+    description: Scenario for testing models on solving coreference resolution problems.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: multiple-choice
+      what: contemporary Chinese literary works
+      who: n/a
+      when: n/a
       language: Chinese
 
   - name: cleva_intent_understanding
     display_name: CLEVA (Chinese) intent understanding
-    description: CLEVA (Chinese) intent understanding
+    description: Tests whether the model could capture the writing intention of the authors after reading an article.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: multiple-choice
```

Comment:
In general, change "task: multiple-choice" to "task: multiple-choice question answering" throughout.